examples/sgx/00-simple_sgx.py

```diff
@@ -14,9 +14,37 @@
     ''',
     basis = 'ccpvdz',
 )
-mf = sgx.sgx_fit(scf.RHF(mol))
+# Direct K matrix for comparison
+mf = scf.RHF(mol)
+mf.kernel()
+
+# Using SGX for J-matrix and K-matrix
+mf = sgx.sgx_fit(scf.RHF(mol), pjs=False)
 mf.kernel()
 
 # Using RI for Coulomb matrix while K-matrix is constructed with COS-X method
 mf.with_df.dfj = True
 mf.kernel()
```

Comment:
It's a bit confusing that you say COS-X here when it's SGX in the previous example. I thought COS-X has P junctions?

Reply:
Agreed, I fixed this.